maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -387,7 +388,7 @@ void cli( CliRequest cliRequest )
 
             if ( configFile.isFile() )
             {
-                for ( String arg : new String( Files.readAllBytes( configFile.toPath() ) ).split( "\\s+" ) )
+                for ( String arg : Files.readAllLines( configFile.toPath(), Charset.defaultCharset() ) )
```

Comment:
Note: I have retained the original file encoding because `jvm.config` is subject to the same platform encoding. We can consider making `maven.config` UTF-8 only.